cli/command/cli.go

```diff
@@ -200,7 +200,7 @@ func (cli *DockerCli) Initialize(opts *cliflags.ClientOptions) error {
 
 		// if server version is lower than the current cli, downgrade
 		if versions.LessThan(ping.APIVersion, cli.client.ClientVersion()) {
-			cli.client.UpdateClientVersion(ping.APIVersion)
+			cli.client.NegotiateAPIVersionPing(ping)
```

Comment:
`NegotiateAPIVersionPing()` does the minimum version check as well, so the previous `if ping.APIVersion == ""` can be removed.